frame/system/src/lib.rs

```diff
@@ -633,13 +666,55 @@ impl Default for InitKind {
 	}
 }
 
+/// Reference status; can be either referenced or unreferenced.
+pub enum RefStatus {
+	Referenced,
+	Unreferenced,
+}
+
 impl<T: Trait> Module<T> {
 	/// Deposits an event into this block's event record.
 	pub fn deposit_event(event: impl Into<T::Event>) {
 		Self::deposit_event_indexed(&[], event.into());
 	}
 
-		/// Deposits an event into this block's event record adding this event
+	/// Increment the reference counter on an account. `_context` is a RefContext value identifying
```

Comment:
```suggestion
	/// Increment the reference counter on an account. `_context` is a RefContext value identifying
```

`_context`? Docs are stale?